logparser/devices/inputdevices.py

```diff
@@ -124,6 +124,9 @@ def print_progress(self, threshold=0, decimals=1, barLength=100):
         # Based on @Greenstick's reply (https://stackoverflow.com/a/34325723)
         iteration = self.stream.tell()
         total = self.file_size
+        if total == 0:
```

Comment:
I think that reading "if not total" is not intuitive. It make sense to compare against 0.